demos/using-dev-build/README.md

```diff
@@ -1,16 +1,63 @@
 # Integrating with Internet Identity
 
-This shows how to integrate and test a project with Internet Identity. This uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors).
+This shows how to integrate and test a project with Internet Identity. This uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors) of Internet Identity and the [agent-js](https://github.com/dfinity/agent-js) library.
 
-## Running the tests
+## Usage
 
 The following commands will start a replica, install the development Internet Identity canister, and run the test suite:
 
 ```bash
-$ dfx start # in a different terminal
-$ dfx deploy --no-wallet --argument '(null)'
+$ dfx start --background --clean
 $ npm ci
-$ npm run test
+$ dfx deploy --no-wallet --argument '(null)'
+```
+
+At this point, the replica (for all practical matters, a local version of the Internet Computer) is running and three canisters have been deployed:
+
+- `internet_identity`: The development version of Internet Identity.
+- `webapp`: A tiny webapp that calls out to the `internet_identity` canister for identity (anchor) creation and authentication, and that then calls the `whoami` canister (see below) to show that the identity is valid. You'll find the source of the webapp in [`index.html`](./index.html) and [`index.js`](./index.js).
+- `whoami`: A simple canister that checks that calls are authenticated, and that returns the "principal of the caller". The implementation is terribly simple:
+  ```motoko
+  actor {
+      public query ({caller}) func whoami() : async Principal {
+          return caller;
+      };
+  };
+  ```
+  On the IC, principals are basically the "identity" of someone performing a request or "call" (hence "caller"), and every call must be authenticated — i.e., must have a valid principal. When using Internet Identity you are using [self-authenticating principals](https://smartcontracts.org/docs/interface-spec/index.html#principal), which is a very fancy way of saying that you have a private key on your laptop (hidden behind TouchID, Windows Hello, etc) that your browser uses to sign and prove that you are indeed the person issuing the calls to the IC. If the IC actually lets the call (request) through to the `whoami` canister, it means that everything checked out, and the `whoami` canister just responds with the information the IC adds to requests, namely your identity (principal).
```

Comment:
Can you split it in two paragraphs? Maybe at "the calls to the IC. (break) If the IC actually"

Reply:
Done